apps/betterangels-backend/clients/tests/test_mutations.py

```diff
@@ -681,6 +690,77 @@ def test_delete_client_document(self) -> None:
             "The document should have been deleted from the database.",
         )
 
+    def test_update_client_document_success(self) -> None:
```

Comment:
you can call this 
```suggestion
    def test_update_client_document(self) -> None:
```
 by default, we assume that a test is verifying the "happy" path